account_credit_control/models/res_partner.py

```diff
@@ -51,6 +52,20 @@ class ResPartner(models.Model):
     manual_followup = fields.Boolean(
         string='Manual Followup',
     )
+    credit_control_notes = fields.Char(
+        help="These are notes related to Credit Control for this partner",
+    )
+    credit_control_email = fields.Char(
+        help="This is the email used in place of standard email to manage"
+             " credit control"
+    )
+
+    @api.model
+    def _get_default_credit_policy(self):
+        policy_obj = self.env['credit.control.policy']
+        default_policy = policy_obj.search([('default_on_partner', '=', True)])
+        if default_policy:
```

Comment:
do we should take the do_nothing field into account.

Reply:
Hi @houssine78, as that field is on the policy, it does not avoid getting a default policy on the partner

Reply:
@rousseldenis my understanding of this field is to set it at True when the policy become obsolete. as it obsolete does it should be taken as default ? I mean people could forget to unset it on a partner. The idea is why it should continue to be returned as default if it's obsolete

Reply:
@houssine78 The field 'default_on_partner' is on the policy not on the partner. Thus forgetting to unset it isn't really a concern imo.

Reply:
ok it's not blocking anyway. ok to move on as it is.